docs/rules/symbol-description.md

```diff
@@ -1,17 +1,29 @@
 # Require/Disallow Symbol Description (symbol-description)
 
-The `Symbol` constructor may have optional description:
+The `Symbol` function may have optional description:
 
 ```js
 var foo = Symbol("foo");
 
 var someString = 'bar';
 var bar = Symbol(someString);
+```
+
+
+Using `description` promotes easier debugging since when symbol is casted to string the description is used:
```

Comment:
I'd remove the part about casting, as it's not technically accurate. Just say the description is present in debuggers.